{"uuid":"7a2d1b63-b0b1-4dc3-96cd-bfb67ec6d35f","historyId":"a2890ba80645f1606a2bf7f1ca8e3bc3","fullName":"com.mas_allure.tests.FaceBookLoginTest.facebookLoginTest","labels":[{"name":"package","value":"com.mas_allure.tests.FaceBookLoginTest"},{"name":"testClass","value":"com.mas_allure.tests.FaceBookLoginTest"},{"name":"testMethod","value":"facebookLoginTest"},{"name":"parentSuite","value":"mas_allure by packages"},{"name":"suite","value":"com.mas_allure.tests"},{"name":"subSuite","value":"Facebook login test"},{"name":"host","value":"LAPTOP-O2PFH35H"},{"name":"thread","value":"19120@LAPTOP-O2PFH35H.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Invalid username and password login test"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"FB Login Tests"},{"name":"severity","value":"blocker"}],"links":[],"name":"Invalid Login Scenario with wrong username and password.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Login failed : Test failed expected [true] but found [false]","trace":"java.lang.AssertionError: Login failed : Test failed expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertTrue(Assert.java:44)\r\n\tat com.mas_allure.tests.FaceBookLoginTest.facebookLoginTest(FaceBookLoginTest.java:38)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Test Description: Login test with wrong username and wrong password.","steps":[{"name":"Enter Username selvaraj","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"email","value":"selvaraj"}],"start":1636598711062,"stop":1636598711141},{"name":"Enter Password Password@741","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"password","value":"Password@741"}],"start":1636598711142,"stop":1636598711242},{"name":"Click SignIn button","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1636598711242,"stop":1636598711325}],"attachments":[],"parameters":[],"start":1636598710165,"stop":1636598740834}